controller/internal/enforcer/applicationproxy/protomux/protomux.go

```diff
@@ -175,9 +175,11 @@ func (m *MultiplexedListener) Serve(ctx context.Context) error {
 		case <-m.shutdown:
 			return nil
 		default:
+
 			c, err := m.root.Accept()
 			if err != nil {
-				return err
+				zap.L().Error("error from Accept", zap.Error(err))
```

Comment:
please add a code comment here why we are not failing anymore (Amit told to me that this is an intermittent error on Windows and needs retry, so that explanation would be nice to have)